system/api/server.go

```diff
@@ -9,4 +9,6 @@ func Run() {
 	http.HandleFunc("/api/content", Record(CORS(Gzip(contentHandler))))
 
 	http.HandleFunc("/api/content/external", Record(CORS(externalContentHandler)))
+
+	http.HandleFunc("/api/content/update", Record(CORS(updateContentHandler)))
```

Comment:
Good point re: HTML form not having support for anything other than GET/POST... 

So, maybe the best move is to keep GET/POST as the only verbs, and set the endpoints to: 

`/api/content(s)` (only GET)
`/api/content/add` or `/api/content/create`? (only POST)
`/api/content/update` (only POST)
`/api/content/delete` (only POST)

Then redefine the API interfaces to:
`Externalable` becomes `Addable` (and it's `Accept` method becomes `Add`)
`Updateable` stays the same
`Deleteable` is created with an `/api/content/delete` route, and has a `Delete` method

Along with this, the default implementations for `Item.Hookable` would need to change to reflect the new names. `api.Trustable` and `editor.Mergeable` would stay the same, as would their relative `Item.Hookable` hooks.

My goal in doing any of this is to just define a pattern so future changes / additions to the API have something clear to follow, and that Ponzu users can infer how things work without memorizing nuance. 

These would be breaking changes, and though we are in a pre-1.0 version, I think these need to be carefully considered as with all breaking changes. However, the importance of establishing a clean, consistent API is definitely more important to avoiding a breaking change before 1.0 -- so now's the time!